test/json.js

```diff
@@ -74,7 +74,8 @@ test('catches errors on invalid non-200 responses', async t => {
 });
 
 test('should have statusCode in err', async t => {
-	const err = await t.throws(got(`${s.url}/non200-invalid`, {json: true}), got.ParseError);
+	const err = await t.throws(got(`${s.url}/non200-invalid`, {json: true}));
+	t.is(err.constructor, new got.ParseError(new Error(), 500, {}, '').constructor);
```

Comment:
Fair question. Too long ago to remember what the issue was. Fairly sure I tried. But maybe I wrongly assumed `t.is` would run into the same problem as `t.throws`. I'll try tonight!